README.md

```diff
@@ -1,76 +1,70 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Getting Started with Create React App
 
-# Project Name
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Another suggestion 😁 Kindly tell us a little bit more about the project! What did you learn or which is the intention of the app? Don't forget that _README file should be descriptive_